### ISSUE TITLE
Bump the lates tensorflow version

### ISSUE THREAD

**Issue by samsja** [docarray/docarray#1776]

**Comment by JohannesMessner** [comment]
I don't think we have a tf version specified at all right now, we just install a certain version in the test CI.
If we now specify this new version that would imply dropping support for all older versions, right?

**Comment by samsja** [comment]
>  If we now specify this new version that would imply dropping support for all older versions, right?

yes 

> I don't think we have a tf version specified at all right now
We don't have a tensorflow version specified because proto version was caped on tensorflow.

This would allow us to have proper dependency management for tensorflow

**Comment by JoanFM** [comment]
But here u are proposing only to cap the TF version to a minimum version? If we do not have any reason not to support the previous versions, what would be the benefit?

**Comment by samsja** [comment]
> But here u are proposing only to cap the TF version to a minimum version? If we do not have any reason not to support the previous versions, what would be the benefit?

benefit would be that tensorflow could be registered as a dependency in docarray.

As of today no package manager know that docarray can have tensorflow has a dependecy

**Comment by JohannesMessner** [comment]
> > But here u are proposing only to cap the TF version to a minimum version? If we do not have any reason not to support the previous versions, what would be the benefit?
> 
> benefit would be that tensorflow could be registered as a dependency in docarray.
> 
> As of today no package manager know that docarray can have tensorflow has a dependecy

That seems more like a benefit to our dev workflow rather than a user benefit.
Yes, it would be nice for them to do `pip install docarray[tf]`, but in return making it incompatible with every but the very newest tf version doesn't seem like a good tradeoff to me.

**Comment by samsja** [comment]
well not only our dev workflow but the dev workflow of our user ...

But yeah I see the argument, lets keep it like this then, but in 6 month probably we should reconsider
